CHANGELOG.md

```diff
@@ -4,3 +4,6 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Checkout the CHANGELOG.md of each package …
+
```

Comment:
Wrong file, please use the CHANGELOG.md in the package as described in the line above